### ISSUE TITLE
Add internationalization for aria-labels

### ISSUE THREAD

**Issue by woutzah** [lhz516/react-h5-audio-player#133]
Hello there,

I would like to suggest a new feature where it would be possible to translate the component, more specific change the aria-labels of not only the custom icons, but also the surrounding buttons for example.

![image](https://user-images.githubusercontent.com/24385658/140739749-03d57dbd-1eef-4c9b-9f10-c539b9d335ee.png)
 
Now it would play both the english and the dutch version for example because the button aria-label is not accessible.

Keep up the good work!

**Comment by lhz516** [comment]
This is definitely on the roadmap. Since an issue is already opened, I'll take this as priority,

**Comment by woutzah** [comment]
Thanks for the reply and priority! If you need additional info or have questions, feel free to ask! Keep up the good work!

**Comment by lhz516** [comment]
Done and published [2.8.0](https://github.com/lhz516/react-h5-audio-player/releases/tag/v3.8.0)
